kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/consumer/KafkaSubscriber.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2010-2023. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.extensions.kafka.eventhandling.consumer;
+
+import org.apache.kafka.clients.consumer.Consumer;
+
+/**
+ * Used by {@link org.axonframework.extensions.kafka.eventhandling.consumer.streamable.StreamableKafkaMessageSource} and {@link org.axonframework.extensions.kafka.eventhandling.consumer.subscribable.SubscribableKafkaMessageSource} to subscribe a {@link Consumer}
+ * to topic(s).
+ *
+ * @author Ben Kornmeier
+ * @since 4.8.0
+ */
+public interface KafkaSubscriber {
```

Comment:
Although it subscribes to Kafka-specifics, I do think that calling this the `TopicSubscriber` makes more sense. What do you think, @bkornmeierpax8?

Reply:
I agree. Yeah KafkaSubscriber is a bit redundant, giving the context of the project, will make it `TopicSubscriber` instead.